.github/workflows/saucelabs.yaml

```
@@ -0,0 +1,40 @@
+name: Integration tests (via Sauce Labs)
+
+# Our Sauce Labs account only allows 5 tests to run at a time
+concurrency: saucelabs
+
+on:
+  pull_request:
+
+jobs:
+  test-saucelabs:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Read node version from .nvmrc
+        id: nvm
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+
+      - name: "Setup Node v${{ steps.nvm.outputs.NVMRC }}"
+        uses: actions/setup-node@v2
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Run integration tests
+        env:
+          SAUCE_ENABLED: true
+          SAUCE_BUILD_NUMBER: "${{ github.run_id }}"
+          SAUCE_USERNAME: "${{ secrets.SAUCE_USERNAME }}"
+          SAUCE_ACCESS_KEY: "${{ secrets.SAUCE_ACCESS_KEY }}"
+        run: |
+          npm run wdio:test
```

Any reason this is a block? Given it's a single command, it'd be more consistent to put it on the same line.

```suggestion
        run: npm run wdio:test
```

---

I found it made it easier to separate it visually from the big block of env declarations above, but I'm not too fussed.

---

Could you flip `env` and `run` around? It's definitely not a deal breaker – happy to leave as is, just a potential source of confusion for folks that aren't as familiar with YAML.